app/components/system_message/index.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import CombinedJoinLeaveMessage from '../combined_joinleave_message/';
+import ChannelSystemMessage from '../channel_system_message/channel_system_message';
+import {Posts} from 'mattermost-redux/constants';
+
+
+export default class SystemMessage extends React.PureComponent {
+
+    render() {
+        const {postType, navigator, theme, textStyles} = this.props;
+        return (postType === Posts.POST_TYPES.PURPOSE_CHANGE || postType === Posts.POST_TYPES.HEADER_CHANGE ||
```

Comment:
Can you change these to regular if/else statements? We shoudl save ternary statements for simpler cases